nncf/config/schema.py

```diff
@@ -735,8 +735,12 @@ def with_attributes(schema: Dict, **kwargs) -> Dict:
                                                                description="Number of training steps to estimate"
                                                                            "pruned model accuracy."),
                                 "max_pruning": with_attributes(_NUMBER,
-                                                               description="Maximum possible pruning level for "
-                                                                           "the model to train LeGR algo on it."),
+                                                               description="Pruning level for the model to train"
+                                                                           " LeGR algorithm on it. If you want to use"
+                                                                           " learned ranking for multiple pruning"
```

Comment:
I don't understand the statement. Maybe it can be rephrased or even removed? Up to you

Reply:
This description was rephrased